touch_element/test_apps/sdkconfig.ci.opt_o0

```diff
@@ -0,0 +1 @@
+CONFIG_COMPILER_OPTIMIZATION_NONE=y
```

Comment:
At the moment the `.ci` files are not taken into consideration in our CI. (point for future enhacement)

Reply:
OK, got it!